app/assets/stylesheets/overlays/_activity-feed-overlay.scss

```diff
@@ -14,9 +13,11 @@
   font-size: 17px;
 }
 
-.activity-feed-overlay-timestamp {
+td.activity-feed-overlay-timestamp {
```

Comment:
I'm no CSS expert but I believe Hound's right here. @nummi any thoughts?

Is it possible to avoid using `td` here?